### PR TITLE
Virtual appliance docs page updates

### PR DESCRIPTION
**Pull request by hflynn**
Decoupling work (https://trello.com/c/ABw65IOY/62-vm-downloads-page) means the Virtual appliance will no longer be available from the main downloads page for 5.1.3. This updates the docs to point users to the new page.

### CODE REVIEW

File: omero/users/virtual-appliance.txt [hunk 1]
```diff
@@ -76,9 +79,8 @@ screenshot:
 Download the OMERO.server virtual appliance
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The virtual appliance can be downloaded from the 
-:downloads:`OMERO download page <>` and should have a filename 
-similar to omero-vm-|release|.ova.
+The virtual appliance has its own :downloads_va:`download page <>` and will
+have a filename similar to omero-vm-|release|.ova.
```
Comment (sbesson) [inline omero/users/virtual-appliance.txt]: OMERO.va-|release| ?

**Comment by hflynn** [comment]
Also see https://trello.com/c/LIz9a2t3/349-virtual-appliance-doc for changes in 2nd commit

**Comment by hflynn** [comment]
@pwalczysko can also add the memory thing and your screenshot if you give me the details of the workflow

**Comment by pwalczysko** [comment]
You can increase the importing speed of the Virtual Appliance server by increasing the Base Memory allocation for the Appliance in you Virtual Box. 
For this, right-click on the Omero Virtual Appliance in the Virtual Box and select ```Settings``.

screen shot 2015-07-17 at 17 32 33

Go to ``System`` tab 

screen shot 2015-07-17 at 17 32 46

and increase the ``Base memory`` by typing into the text box.

Before

screen shot 2015-07-17 at 17 32 56

After

screen shot 2015-07-17 at 17 33 12

After this, restart your OMERO Virtual Appliance (in case it was running already).

**Comment by pwalczysko** [comment]
Happy with the changes. Ready to merge.

**Comment by hflynn** [comment]
@pwalczysko can you just sign off on the extra commit as well please? Staged at https://www.openmicroscopy.org/site/support/omero5.1-staging/users/va-troubleshooting.html#memory-allocation

**Comment by pwalczysko** [comment]
Last commit looks good. All as expected.

**Comment by hflynn** [comment]
https://www.openmicroscopy.org/site/support/omero5.1-staging/users/va-troubleshooting.html#memory-allocation -> happy now @joshmoore ?

**Comment by joshmoore** [comment]
:smile:, yes, thank you!

**Pull request merged**